sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.1.1 (Unreleased)
+## 12.1.1 (2020.03)
```

Comment:
@ljian3377@gmail.com I remember you fixed another issue related to SAS, please involve here? optional field in accessIdentifier?

Reply:
It didn't change user side API provided by our convinience layer.